src/core/linear_solver/src/preconditioner/4C_linear_solver_preconditioner_muelu.hpp

```diff
@@ -77,49 +77,6 @@ namespace Core::LinearSolver
     Teuchos::RCP<MueLu::Hierarchy<Scalar, LocalOrdinal, GlobalOrdinal, Node>> H_;
 
   };  // class MueLuPreconditioner
-
-  /*! \brief MueLu preconditioner for blocked linear systems of equations for contact problems
-   * in saddlepoint formulation.
-   *
-   *  T. A. Wiesner, M. Mayr, A. Popp, M. W. Gee, W. A. Wall: Algebraic multigrid methods for
```

Comment:
Should we keep this citation somewhere in the description of the `MueLu` preconditioner documentation above?